src/stdune/char.mli

```diff
@@ -0,0 +1,9 @@
+include module type of struct
+  include Stdlib.Char
+end
+
+(** Check if a character belongs to the set [{'0'..'9'}]. *)
+val is_digit : t -> bool
+
+(** Check if a character belongs to the set [{'0'..'9', 'a'..'f'}]. *)
```

Comment:
`'A'..'F'` are generally considered hex as well, so this behavior seems surprising to me.
(sorry for this drive-by comment, I offer no opinion on what the actual semantics in shared-cache should be)

Reply:
Yep, I was actually also uneasy about this. I'm happy to rename this function to `is_lowercase_hex` to make this clear. Our only current use case is checking whether a directory has a two-character hex name that we use in the cache, and we use lowercase characters, so it seems best to filter out strange names with uppercase characters.

Reply:
`is_lowercase_hex` sounds good to me.

Reply:
Done.